### ISSUE TITLE
terraform apply --auto-approve fails while downloading cert-manager

### ISSUE THREAD

**Issue by alphaWizard** [rancher/quickstart#141]
I am running this command within quickstart/azure. This error comes after cluster is created.

Error: failed to download "https://charts.jetstack.io/charts/cert-manager-v0.15.1.tgz" (hint: running `helm repo update` may help)

  on ..\rancher-common\helm.tf line 4, in resource "helm_release" "cert_manager":
   4: resource "helm_release" "cert_manager" {

**Comment by bashofmann** [comment]
I could not replicate this and it works for me. Are you maybe behind some kind of HTTP proxy when accessing the internet from your local machine that blocks this?
What happens if you add the Jetstack Helm repo locally with the Helm CLI?
```
helm repo add jetstack https://charts.jetstack.io
```

**Comment by k-harker** [comment]
I was getting the same error `Error: failed to download "https://charts.jetstack.io/charts/cert-manager-v1.0.4.tgz"`.  I had to run the following on my local machine to get it to work.  

```
helm repo add jetstack https://charts.jetstack.io
helm repo update
```

I haven't used terraform before, so I didn't realize that it was running helm from my local machine.

**Comment by bashofmann** [comment]
https://github.com/rancher/quickstart/pull/185 changed the cert-manager installation so that this specific error can not happen anymore.

**Comment by bashofmann** [comment]
Upstream issue: https://github.com/hashicorp/terraform-provider-helm/issues/630

**Comment by bashofmann** [comment]
This seems to have been fixed since version 2.0.0 of the helm terraform provider (https://github.com/hashicorp/terraform-provider-helm/blob/main/website/docs/guides/v2-upgrade-guide.markdown#removal-of-the-helm_repository-data-source) and I can't reproduce this anymore.
